src/reducers/rootReducer.ts

```diff
@@ -7,4 +11,6 @@ const rootReducer = combineReducers({
 
 export type RootState = ReturnType<typeof rootReducer>;
 
+export const useSelector: TypedUseSelectorHook<RootState> = useReduxSelector;
```

Comment:
Why this should be imported and re-exported? is TS complaining about the type so you have to type it before using it?

Reply:
Yes. This is being imported and also typed with TypedUseSelectorHook, so it can be reused with the right type of the state.